layouts/partials/footer.ejs

```diff
@@ -16,6 +16,7 @@
 
   <p class="help-text"><%- template('footer.helpText') %></p>
 </div>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/typeahead.js/0.11.1/typeahead.jquery.min.js"></script>
```

Comment:
It looks like all the other Library scripts are in the head (and we just haven't audited this in a while). I know it's beyond the scope of this particular change, but I think it would be worthwhile for us to be consistent about where scripts are included. This script seems like it is closer to the right place than the existing ones. What do you think about moving the other includes so they can all be together? My hunch is that this wouldn't break anything but would be curious to see what you find.

Reply:
Yeah — I initially had it in the `head` but since it's technically not necessary for the page to paint I wanted to make sure it wasn't blocking. That said, CDNJS has extremely aggressive caching, so I wouldn't be too worried about putting it back